sbi/neural_nets/density_estimators/nflows_flow.py

```diff
@@ -15,8 +14,10 @@ class NFlowsFlow(DensityEstimator):
     wrap them and add the .loss() method.
     """
 
-    def __init__(self, net: Flow, condition_shape: torch.Size) -> None:
```

Comment:
Should we keep the `__init__` here to ensure that `net` is an `nflows.flows` `Flow`?

Reply:
Add it back